README.md

```diff
@@ -239,23 +247,10 @@ file you can `symlink` it with `.env.test`. For example you could run
 commands `ln` and `unlink` for more info.)
 
 ### Prometheus server
-Prometheus metrics can be enabled by running sidecar with the following flag :
-
-```bash
-yarn start --prometheus
-```
-
-You can also define a custom port by running :
-
-```bash
-yarn start --prometheus --prometheus-port=<YOUR_CUSTOM_PORT>
-```
 
-You can also expand the metrics tracking capabilities to include query params by running:
+Prometheus metrics can be enabled by running sidecar with the following env configuration: `SAS_METRICS_ENABLED`=true
 
-```bash
-yarn start --prometheus --prometheus-queryparams
-```
+You can also expand the metrics tracking capabilities to include query params by adding to the env configuration: `INCLUDE_QUERYPARAMS`=true
```

Comment:
I set `INCLUDE_QUERYPARAMS=true` in the `.env` file and I still get `INCLUDE_QUERYPARAMS=false` when I start Sidecar.

Reply:
yes sorry its SAS_METRICS_INCLUDE_QUERYPARAMS